mrjob/examples/mr_nick_nack_input_format.py

```diff
@@ -0,0 +1,11 @@
+from mrjob.examples.mr_nick_nack import MRNickNack
+
+
+class MRNickNackWithHadoopInputFormat(MRNickNack):
+
+    HADOOP_INPUT_FORMAT = 'nicknack.ManifestTextInputFormat'
+    HADDOP_OUTPUT_FORMAT = None
```

Comment:
setting `HADOOP_OUTPUT_FORMAT` to `None` doesn't do anything because that's the default. Also you misspelled it.

Reply:
Well but it inherits from MRNickNack, which sets `HADOOP_OUTPUT_FORMAT = 'nicknack.MultipleValueOutputFormat'`

However, I guess the fact that everything still ran is proof that it doesn't matter really!

Reply:
Oh... good point. Okay, please correct the typo.